WorkloadManagementSystem/Agent/StalledJobAgent.py

```diff
@@ -443,10 +443,10 @@ def __checkLoggingInfo( self, jobID, jobDict ):
       if not startTime or startTime == 'None':
         startTime = jobDict['SubmissionTime']
 
-    if type( startTime ) in types.StringTypes:
+    if isinstance(startTime, basestring):
       startTime = fromString( startTime )
       if startTime == None:
-        self.log.error( 'Wrong timestamp in DB', items[3] )
+        self.log.error( 'Wrong timestamp in DB', startTime )
```

Comment:
You want to print the original `startTime` string. Because here `startTime == None`